app/src/main/java/com/example/fivenightsescape/player/Player.kt

```diff
@@ -1,18 +1,22 @@
 package com.example.fivenightsescape.player
 
 import android.location.Location
+import android.widget.TextView
 import androidx.lifecycle.ViewModel
 import com.google.android.gms.maps.model.LatLng
 
-const val PLAYER_HEALTH = 3
+private const val PLAYER_HEALTH = 3
 
-class Player(var position: LatLng): ViewModel() {
+class Player(var position: LatLng, val healthBar: TextView): ViewModel() {
     private var health: Int = PLAYER_HEALTH
+
     var location: Location = Location("")
 
     init {
         this.location.latitude = this.position.latitude
         this.location.longitude = this.position.longitude
+
+        this.healthBar.text = this.health.toString()
```

Comment:
Should be: (for both / one below)
```kt
"Health: " + this.health.toString()
```
as user won't know what numbers stand for.
![Screenshot 2024-01-18 001035](https://github.com/PKrystian/FiveNightsEscape/assets/97467746/90c8dcad-1555-44d7-bc7c-e09354705e41)